### PR TITLE
Translate team-goals

### PR DESCRIPTION
**Pull request by Fridolin012**
## Type of PR

- [X] Translation update / creation
- [ ] Original file creation
- [ ] Original file content (not typo) update
- [ ] Original file typo update

## Checks

If you have changed the content (/ meaning) of an original file, please confirm:

- [ ] Yes, I have copied the updated section to all translations of this file manually.

## Notes

Please enter additional information here.

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+Setze wöchentliche Nachrichtenziele für deine Teammitglieder.
+
+<ModuleOverview moduleName="team-goals" />
+
+## Funktionen {#features}
+
+* Setze ein Nachrichtenziel für deine Teammitglieder.
+* Der Bot wird jede Woche überprüfen, ob die Teammitglieder das Nachrichtenziel erreicht haben.
+* Überprüfe mit [Befehlen](#commands) deinen aktuellen Fortschritt und Erfolge in der Vergangenheit.
+* Sende Nutzern jede Woche über PNs ihr Ergebnis oder in einen konfigurierten Kanal.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:12]:
```suggestion
* Sende Nutzern ihr Ergebniss jede Woche per PN oder in einen konfigurierten Kanal.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+1. Wähle (oder erstelle) eine Rolle. Die Aktivität von Nutzern mit dieser Rolle wird gewertet.
+2. [Konfiguriere das Modul](#configuration).
+3. [Setze bitte Berechtigungen](./../../slash-commands) für Slash-Befehle, um unerlaubte Nutzung zu unterbinden.
+4. Gebe Mitgliedern, deren Aktivität ausgewertet werden soll, die Rolle.
+
+## Nutzung {#usage}
+
+* Die Anforderungen werden alle 7 Tage nach Aktivierung des Moduls automatisch [ausgewertet](#module-terms).
+* Du kannst immer den Fortschritt von dir oder einem anderem Nutzer mit [`/team-goals progress`](#commands) und die Zielhistorie mit [`/team-goals history`](#commands) einsehen.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:25]:
```suggestion
* Du kannst immer den Fortschritt von dir oder einem anderem Nutzer mit [`/team-goals progress`](#commands) und den früheren Zielverlauf mit [`/team-goals history`](#commands) einsehen.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+## Befehle {#commands}
+
+<SlashCommandExplanation />
+
+| Befehl                                 | Beschreibung                                                                                                                                                                                                                        |
+|----------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `/team-goals progress [user:<Nutzer>]` | Zeigt den Fortschritt (Anzahl der Nachrichten, die zum Erreichen des Ziels gebraucht werden, verbleibende Zeit, …) für das Ziel des aktuellen [Bewertungszeitraums](#module-terms) an (wenn leer, wird dein Fortschritt angezeigt). |
+| `/team-goals history [user:<Nutzer>]`  | Zeigt die Zielhistorie (Anzahl der Nachrichten, Ziel erreicht oder nicht, Prozentsatz der erreichten Ziele) der letzten 10 Wochen des angegebenen Benutzers an (wenn leer, wird dein Fortschritt angezeigt).                        |
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:34]:
```suggestion
| `/team-goals history [user:<Nutzer>]`  | Zeigt den früheren Zielverlauf (Anzahl der Nachrichten, Ziel erreicht oder nicht, Prozentsatz der erreichten Ziele) der letzten 10 Wochen des angegebenen Benutzers an (wenn leer, wird dein Verlauf angezeigt).                        |
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+## Definition von modulspezifischen Begriffen {#module-terms}
+
+* Eine **Auswertung** ist der Zeitpunkt, zu dem ein Nutzer ausgewertet wird. Dies geschieht einmal pro Woche. Der eigentliche         Zeitpunkt ist davon abhängig, wann das Modul zum ersten mal aktiviert wurde - die erste Auswertung wird genau eine Woche
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:38]:
```suggestion
* Eine **Auswertung** ist der Zeitpunkt, zu dem ein Nutzer ausgewertet wird. Dies geschieht einmal pro Woche. Der eigentliche Zeitpunkt ist abhängig davon, wann das Modul zum ersten mal aktiviert wurde - die erste Auswertung wird genau eine Woche
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+## Konfiguration {#configuration}
+
+Diese Konfigurationsdatei ermöglicht es dir, die Funktionsweise des Moduls und das Ziel selbst zu konfigurieren.
+Öffne sie in
+deinem [Dashboard](https://scnx.app/de/glink?page=bot/configuration?query=goal&file=team-goals%7Cconfig).
+
+| Feld                                                     | Beschreibung                                                                                                                                                                                                                                  | 
+|----------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Team-Rollen                                              | Diese Rollen werden jede Woche hinsichtlich ihres Fortschritts zum Ziel [ausgewertet](#module-terms). Nur von Nutzern mit diesen Rollen werden die Nachrichten gezählt.                                                                       |
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:50]:
```suggestion
| Team-Rollen                                              | Diese Rollen werden jede Woche hinsichtlich ihres Fortschritts zum Ziel [ausgewertet](#module-terms). Es werden nur die Nachrichten von Nutzern mit dieser Rolle gezählt.                                                                       |
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+| Nachrichten in Kanal statt per PN schicken               | Wenn aktiviert, werden Teammitglieder ihre [Auswertung](#module-terms) in einem Kanal statt per PN erhalten.                                                                                                                                  |
+| Kanal, in welchen die Nachrichten gesendet werden sollen | *Nur sichtbar, wenn "Nachrichten in Kanal statt per PN schicken" aktiviert ist.*<br/>Die [Auswertung](#module-terms) wird in diesen Kanal statt per PN gesendet.                                                                              |
+| Ignorierte Channel                                       | Nachrichten von Nutzern mit mindestens einer der Teammitglieder-Rollen werden in diesen Kanälen nicht zu ihrem Ziel hinzugerechnet.                                                                                                           |
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:56]:
```suggestion
| Ignorierte Kanäle                                       | Nachrichten von Nutzern mit mindestens einer der Teammitglieder-Rollen werden in diesen Kanälen nicht zu ihrem Ziel hinzugerechnet.                                                                                                           |
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+## Fehlerbehebung {#troubleshooting}
+
+<details>
+  <summary>Von meinem Team gesendete Nachrichten werden nicht richtig gezählt</summary>
+  Bitte stelle sicher, dass
+  <ul>
+    <li>der Bot Zugriff auf jeden Kanal hat, von welchem Nachrichten gezählt werden sollen und dass der Kanal kein <a href="#configuration">ignorierter Kanal</a> ist.</li>
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:64]:
```suggestion
    <li>der Bot Zugriff auf jeden Kanal hat, in welchem Nachrichten gezählt werden sollen, und dass der Kanal kein <a href="#configuration">ignorierter Kanal</a> ist.</li>
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+    <li>der Nutzer mindestens eine der <a href="#configuration">konfigurierten Teammitglieder-Rollen</a> hat. Es werden nur Nachrichten gezählt, die gesendet wurden, während mindestens eine Teammitglieder-Rolle besessen wurde - Nachrichten, die gesendet wurden, bevor Nutzer die Teammitglieder-Rolle hatten, werden nie gezählt.</li>
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:65]:
```suggestion
    <li>der Nutzer mindestens eine der <a href="#configuration">konfigurierten Teammitglieder-Rollen</a> hat. Es werden nur Nachrichten gezählt, die von Nutzern gesendet wurden, welche während dem Senden mindestens eine Teammitglieder-Rolle besessen haben - Nachrichten, die gesendet werden, bevor ein Nutzer die Teammitglieder-Rolle hat, werden nicht gezählt.</li>
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+<details>
+  <summary>Die Auswertung funktioniert nicht richtig (z.B. werden Auswertungsnachrichten nicht gesendet)</summary>
+  Bitte stelle sicher, dass
+   <ul>
+    <li>der Bot die Berecchtigung hat, Nachrichten in den <a href="#configuration">konfigurierten Kanal</a> zu senden, wenn aktiviert.</li>
+    <li>deine Teammitglieder mindestens eine der <a href="#configuration">konfigurierten Team-Rollen</a> besitzen.</li>
+    <li>die <a href="#configuration">konfigurierte Nachricht</a> nicht inkorrekt ist.</li>
+  </ul>
+  Selbst wenn keine Auswertung gesendet wurde, kannst du immer die Zielhistorie mit dem Slash-Befehl <a href="#commands"><code>/team-goals history</code></a> einsehen.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:83]:
```suggestion
  Selbst wenn keine Auswertung gesendet wurde, kannst du immer den früheren Zielverlauf mit dem Slash-Befehl <a href="#commands"><code>/team-goals history</code></a> einsehen.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+</details>
+
+<details>
+  <summary>Kann ich das Modul nutzen, um die Aktivität meiner Nutzer auszuwerten? / SCNX fügt einen "Missbrauch melden"-Knopf an meine Nachrichten an</summary>
+  Eine solche Nutzung verstößt gegen unsere <a href="https://sc-net.work/scnx-tos">Nutzungsbedingungen</a>, da du unsere Software nutzt, um deine mit Spam zu belästigen. Du darfst das Modul nur
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:88]:
```suggestion
  Eine solche Nutzung verstößt gegen unsere <a href="https://sc-net.work/scnx-tos">Nutzungsbedingungen</a>, da du unsere Software nutzt, um deine Nutzer mit Spam zu belästigen. Du darfst das Modul nur
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+
+Die folgenden Daten werden über jeden Nutzer mit einer [konfigurierten Team-Rolle](#configuration) gespeichert, sobald sie eine
+Nachricht gesendet haben ("Nutzer-Eintrag"):
+
+* Die eindeutige Discord-Nutzer-ID
+* Die Anzahl der Nachrichten, die in dem [aktuellen Bewertungszeitraum](#module-terms) gesendet wurden
+* Metadaten zum Eintrag (Datum der Erstellung und der letzten Aktualisierung)
+
+Das folgende wird bei jeder [Auswertung](#module-terms) für jeden Nutzer gespeichert, der eine der
+[konfigurierten Team-Rollen](#configuration) hat ("Zielhistorie-Eintrag"):
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:105]:
```suggestion
[konfigurierten Team-Rollen](#configuration) hat ("Zielverlauf-Eintrag"):
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+
+* Eine eindeutige Zahl zur Identifizierung des Datenbankeintrags
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:107]:
```suggestion
* Eine eindeutige Nummer zur Identifizierung des Datenbankeintrags
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+* Die eindeutige Discord-Nutzer-ID
+* Die Anzahl der Nachrichten, die sie im [Bewertungszeitraum](#module-terms) gesendet haben
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:109]:
```suggestion
* Die Anzahl der Nachrichten, die der Nutzer im [Bewertungszeitraum](#module-terms) gesendet hat
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+* Das [konfigurierte Ziel](#configuration) des aktuellen [Bewertungszeitraums](#module-terms)
+* Metadaten zum Eintrag (Datum der Erstellung und der letzten Aktualisierung)
+
+Es gibt keine Möglichkeit, den Nutzer-Eintrag oder den Zielhistorie-Eintrag aus der Datenbank zu entfernen.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:113]:
```suggestion
Es gibt keine Möglichkeit, den Nutzer-Eintrag oder den Zielverlauf-Eintrag aus der Datenbank zu entfernen.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 15]
```diff
@@ -0,0 +1,116 @@
+# Teammitglieder-Ziele
+
+Setze wöchentliche Nachrichtenziele für deine Teammitglieder.
+
+<ModuleOverview moduleName="team-goals" />
+
+## Funktionen {#features}
+
+* Setze ein Nachrichtenziel für deine Teammitglieder.
+* Der Bot wird jede Woche überprüfen, ob die Teammitglieder das Nachrichtenziel erreicht haben.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:10]:
```suggestion
* Der Bot überprüft jede Woche, ob die Teammitglieder das gesetze Nachrichtenziel erreicht haben.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 16]
```diff
@@ -0,0 +1,116 @@
...
+* Überprüfe mit [Befehlen](#commands) deinen aktuellen Fortschritt und Erfolge in der Vergangenheit.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:11]:
```suggestion
* Mit [Befehlen](#commands) kannst du deinen aktuellen Fortschritt und deinen Zielverlauf in der Vergangenheit überprüfen.
```

**Review by SCDerox — CHANGES_REQUESTED** [review]

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md (hunk 1 above)
Comment (Fridolin012) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:12]: I think it's "Ergebnis".

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md (hunk 6 above)
Comment (Fridolin012) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:56]: I used the term from the dashboard.

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md (hunk 1 above)
Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:12]: updated suggestion

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md (hunk 6 above)
Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:56]: Please follow our established process to report missing translations. It's still a wrong translation and as such needs to be translated here correctly.

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md [hunk 17]
```diff
@@ -0,0 +1,116 @@
...
+  <summary>Wie kann ich die Zeit verändern, wann Nutzer ausgewertet werden?</summary>
+  Es gibt keinen einfachen Weg, das zu tun. Der Auswertungszeitpunkt basiert auf dem genauen Zeitpunkt, zu dem das Modul das erste mal aktiviert wurde (die Auswertung findet jede Woche an diesem Tag zu dieser Zeit statt). Wenn du <i>wirklich</i> einen anderen Auswertungszeitpunkt benötigst, kannst du das Modul deaktivieren, 
+  die <a href="./../../additional-features#reset-module-database">Modul-Datenbank löschen</a> und warten, bis der genaue Zeitpunkt erreicht ist, zu der die Auswertung stattfinden soll, und dann das Modul wieder aktivieren.
+</details>
+
+<details>
+  <summary>Die Auswertung funktioniert nicht richtig (z.B. werden Auswertungsnachrichten nicht gesendet)</summary>
+  Bitte stelle sicher, dass
+   <ul>
+    <li>der Bot die Berecchtigung hat, Nachrichten in den <a href="#configuration">konfigurierten Kanal</a> zu senden, wenn aktiviert.</li>
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:79]:
```suggestion
    <li>der Bot die Berechtigungen hat, Nachrichten in den <a href="#configuration">konfigurierten Kanal</a> zu senden, wenn aktiviert.</li>
```

**Review by SCDerox — CHANGES_REQUESTED** [review]

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md (hunk 17 above)
Comment (Fridolin012) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:79]: It's just one permission, isn't it?

Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/team-goals.md:79]: Yes, I fixed my comment, sorry, still a typo

**Review by SCDerox — APPROVED** [review]

**Comment by SCDerox** [comment]
Thanks for your contribution - you have received a payout as part of our Open-Source Developer Program. You can find details in your SCNX organization payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on scnx.app/help.

**Pull request merged**